SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -60,6 +65,18 @@ class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
 	UFUNCTION(BlueprintPure, Category = "SpatialOS|Offloading", meta = (WorldContext = "WorldContextObject"))
 	static FName GetActorGroupForClass(const UObject* WorldContextObject, const TSubclassOf<AActor> ActorClass);
 
+	/**
+	 * Functionally the same as the native Unreal PrintString but also logs to the spatial runtime.
+	 */
+	UFUNCTION(BlueprintCallable, meta = (WorldContext = "WorldContextObject", CallableWithoutWorldContext, Keywords = "log print spatial", AdvancedDisplay = "2", DevelopmentOnly), Category = "Utilities|String")
```

Comment:
Should we change the Category of these to "Spatial|Utilities" or something?

Reply:
Tested this, made the discoverability worse so didn't go through with the change.